src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/sge/SGEJobManager.java

```diff
@@ -1,162 +0,0 @@
-///*
```

Comment:
We still support SGE. I think the changes in the SGE classes might a lot of clashes!